src/ui/preferences-window/LabelAndControl.swift

```diff
@@ -260,12 +260,14 @@ class LabelAndControl: NSObject {
                 button.setWidth(segmentWidth, forSegment: i)
             }
             if #available(macOS 11.0, *) {
-                if let preference = preference as? SfSymbolMacroPreference {
-                    button.setImage(NSImage(systemSymbolName: preference.symbolName, accessibilityDescription: nil)!, forSegment: i)
+                if let preference = preference as? SfSymbolMacroPreference,
+                   let symbolImage = NSImage(systemSymbolName: preference.symbolName, accessibilityDescription: nil) {
+                    button.setImage(symbolImage, forSegment: i)
```

Comment:
I don't think this part has issues since every single crash is running macOS 10.12 or lower. They shouldn't run code in that section.

Also, I think it's healthy to keep the code crashing here. This helps us identify if an image is missing during development. If we had this condition here, we may ship a release where we the images don't work, and the users will have a degraded experience without images.

Reply:
![QQ_1728720386963](https://github.com/user-attachments/assets/d3abae15-2460-4bcd-9805-c096f209dcd3)

This requires macOS 13+

Reply:
There is no image but text is displayed. I think it is a better way to downgrade the display text here. 

What do you think?

Reply:
Oh! So we add an image on macOS `>= 11` but some images have specific availability, and swift is not protecting us from this. That's a very nice catch.

I don't think it explains the crashes here, since users who crash are on macOS `<= 10.12`, so they shouldn't enter the top condition in the first place. However, we your change definitely will help users on macOS `>= 11 and < 13`. I'll add a comment so we remember later why it's important to not assume the images exist 👍

Reply:
Yes, it's definitely an issue. However, as I said, it should only create crashes in the case were macOS `>= 11 and < 13`. Yet our crashed users are all on macOS 10.12 or 10.11. Weird, no?

Reply:
![QQ_1728723566159](https://github.com/user-attachments/assets/30994ac0-3314-405c-97f4-e2aeeddd70fa)

The version of macOS that the user is having problems with is `12.7.6`, So I think that's the root reason. 
If we don’t downgrade the experience, users won’t be able to use the app at all, which will have a greater impact.

Reply:
> Yes, it's definitely an issue. However, as I said, it should only create crashes in the case were macOS `>= 11 and < 13`. Yet our crashed users are all on macOS 10.12 or 10.11. Weird, no?

You can see the crash version on AppCenter, which is distributed in `12.7.6`,`11.7.10`,`12.7.5`.